models/articles.models.js

```diff
@@ -17,23 +17,29 @@ function selectArticleById(article_id){
 
 
 //task 5&task 11
-function selectArticle(sort_by= 'created_at',order = 'desc'){
+function selectArticle(sort_by= 'created_at',order = 'desc',topic=null){
```

Comment:
no need to define topic as null, if you pass no topic to the function then topic will simply be undefined